index.php

```diff
@@ -20,6 +20,7 @@
     ENDPOINT => function() {
 
         header('Access-Control-Allow-Origin: *');
+        header('Access-Control-Allow-Headers: content-type');
```

Comment:
Can you point me to a good reference on this, would like to understand it better